workflow-examples/src/main/java/com/redhat/parodos/examples/ocponboarding/OcpOnboardingWorkFlowConfiguration.java

```diff
@@ -76,15 +78,24 @@ WorkFlowOption notSupportOption() {
 						.setDescription("Non-Supported Workflow Steps").build();
 	}
 
+	@Bean
+	WorkFlowOption analyzeOption() {
+		return new WorkFlowOption.Builder("analyzeOption", "AnalyzeApplicationAssessment")
```

Comment:
I'd like to suggest adding a class named `AnalysysConstants` or `MTAConstants` that will list the constants used here and reduces the chances of misspelling the keys.
We don't have such a convention yet, but we have that as part of Move2kube and I find it useful.